File: Java/abc020A.java
import java.util.Scanner;

class Main{
  public static void main(String args[]){
    Scanner scan = new Scanner(System.in);
    String n = scan.nextLine();
    if(Integer.parseInt(n) == 1){
      System.out.println("ABC");
    }else{
      System.out.println("chokudai");
    }
  }
}

File: JavaScript/abc020A.js
function Main(input) {
    var a = input.split('\n');
    if( parseInt(a) == 1 ) {
      console.log("ABC");
    }else{
      console.log("chokudai");
    }
  }
  
  Main(require("fs").readFileSync("/dev/stdin", "utf8"));
  